superset/views.py

```diff
@@ -1104,6 +1104,41 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """Used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST'])
+    def store(self):
+        value = request.form.get('data')
+        if not value:
+            return Response(
+                __("Error: data can not be empty"), status=404)
+        base_view = request.form.get('baseView')
+        if not base_view:
+            return Response(
```

Comment:
return json_error_response(__("Error: please provide a base view"))
500 would be better than 404 as it is application error, but not `not found`